app/_kong_plugins/oas-validation/index.md

```diff
@@ -38,6 +38,95 @@ search_aliases:
+
+      If you don't want the plugin to validate the `Etag` with the `If-Match` request header,
+      send the `If-Match` header with a wildcard (`*`) to skip validation.
+
+      For example:
+      ```sh
+      curl http://localhost:8000/example-route \
+        -H 'If-Match:*'
+      ```
+
+related_resources:
+  - text: Event Hooks
+    url: /gateway/entities/event-hook/
 ---
 
-## Overview
+Validate HTTP requests and responses against an OpenAPI Specification.
+
+The plugin supports Swagger v2 and OpenAPI 3.0.x and 3.1.0 specifications with a JSON Schema validator that supports [Draft 2019-09](https://json-schema.org/specification-links#draft-2019-09-(formerly-known-as-draft-8)).
+
+## Supported OpenAPI 3.1.0 specification features
+
+Starting with {{site.base_gateway}} 3.7, the OAS Validation plugin supports the following OpenAPI specification features:
+
+| Category                        | Supported                      | Not supported                                                            |
+|---------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| Request body                    | `application/json`             | `application/xml`<br>`multipart/form-data`<br>`text/plain`<br>`text/xml` |
+| Response body                   | `application/json`             | -                                                                        |
+| Request parameters              | `path`<br>`query`<br>`header`<br>`cookie` | -                                                             |
+| Schema                          | `allOf`<br>`oneOf`<br>`anyOf`  | -                                                                        |
+| Parameter serialization         | `style`<br>`explode `          | -                                                                        |
+
+## Using Event Hooks with OAS Validation
+
+[Event Hooks](/gateway/entities/event-hook/) are outbound calls from {{site.base_gateway}}. 
+With Event Hooks, {{site.base_gateway}} can communicate with target services or resources, letting the target know that an event was triggered. 
+
+For the OAS Validation plugin, Event Hook events can be enabled when a validation fails for:
+* All request parameters, including: URI, header, query parameters, and request body
+* The response body from the upstream service
+
+For example, to configure an Event Hook for the OAS Validation plugin:
+```sh
+curl -i -X POST http://localhost:8001/event-hooks/ \
+  --header "Accept: application/json" \
+  --header "Content-Type: application/json" \
+  --data '
+    {
+      "source": "oas-validation",
+      "event": "validation-failed",
+      "handler": "webhook",
+      "on_change": true,
+      "config": {
+        "url": "$WEBHOOK_URL"
+      }
+    }
+    '
+```
```

Comment:
```suggestion
For example, to configure an Event Hook for the OAS Validation plugin:
{% entity_example %}
type: event_hook
data:
  source: "oas-validation"
  event: "consumers"
  handler: "webhook"
  on_change: true
config:
      "url": "$WEBHOOK_URL"
{% endentity_example %}
```

Will this work?

Reply:
Event hooks can't be configured using decK.